content/millennium/r4/scheduling/slot.md

```diff
@@ -193,6 +193,68 @@ List an individual Slot by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Patch
+
+<%= beta_tag(action: true) %>
+
+Known Issues:
+
+* A 500 Internal Server Error is thrown when an invalid version id is used in the If-Match Header
```

Comment:
Since we are using a helper method to pass in the known issues into the beta tag, any additional markdown is creating issues, so there is no easy way to fix it as of now.